src/components/PostTable.js

```diff
@@ -1,13 +1,141 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import Author from './Author';
+
+const TableWrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding-bottom: 127px;
+  max-width: 786px;
+  width: 100%;
+  margin: 0 auto;
+`;
+
+const PostsTable = styled.table`
+  margin: 1px auto 0;
+  border-collapse: collapse;
+  border-spacing: 0;
+  table-layout: fixed;
+  width: 100%;
+`;
+
+const H2Wrapper = styled.div`
+  margin: 31px auto 10px;
+  width: 100%;
+`;
+
+const H2 = styled.h2`
```

Comment:
Imo it would be better to change the name to something more generic like Headline. Now it's tied to the HTML tag `h2`